components/automate-ui/src/app/ui.component.ts

```diff
@@ -77,5 +80,32 @@ export class UIComponent implements OnInit, AfterViewChecked {
     if (this.chefSessionService.connector && this.userPrefsService.uiSettings.isTimeformatExist) {
       this.store.dispatch(new GetUserPreferences());
     }
+
+    // manual upgrade banner
+    const bannerStorage = localStorage.getItem('manual-upgrade-banner');
+    if (isNull(bannerStorage)) {
+      localStorage.setItem('manual-upgrade-banner', this.booleanToString(true));
```

Comment:
I think this also should be set as 
```ts
localStorage.setItem('manual-upgrade-banner', this.booleanToString(false));
```

Reply:
This is the current behavior, and to minimize the changes to enable the banner I feel the change is not quired.

https://user-images.githubusercontent.com/44021874/157400341-4ff422f2-312d-4327-b55e-7acf05039d33.mov

Reply:
@vivek-yadav  updated the code